mbf_abstract_nav/src/controller_action.cpp

```diff
@@ -191,6 +191,8 @@ void ControllerAction::run(GoalHandle &goal_handle, AbstractControllerExecution
         fillExePathResult(mbf_msgs::ExePathResult::CANCELED, "Controller canceled", result);
         goal_handle.setCanceled(result, result.message);
         controller_active = false;
+        execution.stop();
+        execution.publishZeroVelocity();
```

Comment:
do you know where stop is actually implemented? it's virtual in the AbstractControllerExecution class, but I don't see it on CostmapControllerExecution either. I'm wondering if we should put publishZeroVelocity inside of it instead of here.

Reply:
It's in the abstractExecutionBase class and interrupts the thread.

Reply:
I feel like there's some way this could be more elegant, but I can't think of it, so i'm fine with this.